tendrl/commons/utils/generate_ssh.py

```diff
@@ -0,0 +1,47 @@
+from ansible_module_runner import AnsibleExecutableGenerationFailed
+from ansible_module_runner import AnsibleRunner
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+ANSIBLE_MODULE_PATH = "core/system/user.py"
```

Comment:
Does this module guarantee that the SSH key will be generated if the user already exists?